packages/hemera-nsq/index.js

```diff
@@ -114,15 +113,24 @@ function hemeraNsqStore(hemera, opts, done) {
     hemera.add(
       {
         topic: 'nsq',
-        cmd: 'subscribe',
-        subject: Joi.string().required(),
-        channel: Joi.string().required()
+        cmd: 'subscribe'
       },
-      function(req, cb) {
-        consume(req.subject, req.channel, cb)
+      function(req, reply) {
+        consume(req.subject, req.channel, reply)
       }
     )
 
     done()
   })
+
+  // Gracefully shutdown
+  hemera.ext('onClose', (ctx, done) => {
```

Comment:
One another point called it always `hemera` instead `ctx`.

Reply:
https://github.com/hemerajs/hemera/blob/master/packages/hemera-mongo-store/index.js#L57

Reply:
it might be better if I rewrite it with an empty argument? Like:
```js
hemera.ext('onClose', (_, done) => {
  // ...
})
```
First argument isn't used in this scope.

Reply:
allright